### PR TITLE
Added link how to use MQP on Magento Commerce Cloud

### PR DESCRIPTION
**Pull request by viktym**
## Purpose of this pull request

Updates the [Magento Quality Patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching/mqp.html) topic in the _Software Update Guide_ to link to instructions for using MQP for Magento Commerce projects deployed on the Cloud platform.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/comp-mgr/patching/mqp.html

### CODE REVIEW

File: src/guides/v2.3/comp-mgr/patching/mqp.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ functional_areas:
   - Upgrade
 ---
 
-The [Magento Quality Package (MQP) package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}.
+The [Magento Quality Package (MQP) package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}. How to install and use MQP for Magento Commerce Cloud you can find information in [this article]({{ site.baseurl }}/cloud/project/project-patch.html).
```
Comment (BarnyShergold) [inline src/guides/v2.3/comp-mgr/patching/mqp.md:8]:
```suggestion
The [Magento Quality Package (MQP) package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}. Information on how to install and use MQP for Magento Commerce Cloud can be found in the [Apply Patches]({{ site.baseurl }}/cloud/project/project-patch.html) article.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please also update the PR description to properly describe the PR and add the full link to the page affected - please check other PRs for correct format.

**Review by BarnyShergold — APPROVED** [review]
Can you also add https://devdocs.magento.com/guides/v2.4/comp-mgr/patching/mqp.html to the docs affected as it is symlinked to 2.3. 

As this is minor, I will approve.

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**